File: app/src/main/java/tourtle/ticketing/model/Tiket.java
package tourtle.ticketing.model;


public class Tiket {

    private String idTiket;
    private Integer jumlahOrang;
    private Integer jumlahKendaraan;
    private String hari;
    private String idWisata;
    private String idNegara;
    private String print;
    private Integer totalBiaya;
    private Integer biayaTiket;
    private Integer biayaParkir;

    public Tiket(){

    }



    public void setHari(String hari) {
        this.hari = hari;
    }

    public void setTotalBiaya(int totalBiaya) {
        this.totalBiaya = totalBiaya;
    }

    public void setBiayaTiket(int biayaTiket) {
        this.biayaTiket = biayaTiket;
    }

    public void setBiayaParkir(int biayaParkir) {
        this.biayaParkir = biayaParkir;
    }

    public String getPrint() {
        return print;
    }

    public Tiket(String idTiket, String idWisata , Integer jumlahOrang, Integer jumlahKendaraan, Integer totalBiaya, Integer biayaParkir, Integer biayaTiket,
                 String hari, String idNegara, String print){
        this.idTiket = idTiket;
        this.idWisata = idWisata;
        this.jumlahOrang = jumlahOrang;
        this.jumlahKendaraan = jumlahKendaraan;
        this.totalBiaya = totalBiaya;
        this.biayaParkir = biayaParkir;
        this.biayaTiket = biayaTiket;
        this.hari = hari;
        this.idNegara = idNegara;
        this.print = print;

    }


    public String getHari() {
        return hari;
    }

    public String getIdNegara() {
        return idNegara;
    }

    public Integer getBiayaTiket() {
        return biayaTiket;
    }

    public String getIdWisata() {
        return idWisata;
    }

    public Integer getJumlahKendaraan() {
        return jumlahKendaraan;
    }

    public Integer getBiayaParkir() {
        return biayaParkir;
    }

    public String getIdTiket() {
        return idTiket;
    }

    public Integer getJumlahOrang() {
        return jumlahOrang;
    }

    public Integer getTotalBiaya() {
        return totalBiaya;
    }
}

File: app/src/main/java/tourtle/ticketing2/model/Realm/Tiket.kt
package tourtle.ticketing2.model.Realm

import io.realm.RealmObject
import io.realm.annotations.RealmClass

@RealmClass
open class Tiket: RealmObject() {

    private var _id:Int = 0
    private var idWisata: Int = 0
    private var biayaParkir: Int = 0
    private var biayaTiket: Int = 0
    private var totalBiaya: Int = 0
    private var jalan: Int = 0
    private var rodaDua: Int = 0
    private var rodaEmpat: Int = 0
    private var bus: Int = 0
    private var jumlahKendaraan: Int = 0
    private var jumlahOrang: Int = 0
    private var wisdom: Int = 0
    private var wisman: Int = 0
    private var timestamp: String = ""
    private var taxbanyuwangi: String = ""

    fun setId(id:Int) {
        this._id = id
    }

    fun getId(): Int {
        return _id
    }

    fun setIdWisata(idwisata:Int) {
        this.idWisata = idwisata
    }

    fun getIdWisata(): Int {
        return idWisata
    }

    fun setBiayaParkir(b_parkir:Int) {
        this.biayaParkir = b_parkir
    }

    fun getBiayaParkir(): Int {
        return biayaParkir
    }

    fun setBiayaTiket(b_tiket:Int) {
        this.biayaTiket = b_tiket
    }

    fun getBiayaTiket(): Int {
        return biayaTiket
    }

    fun setTotalBiaya(total:Int) {
        this.totalBiaya = total
    }

    fun getTotalBiaya(): Int {
        return totalBiaya
    }

    fun setJalan(nilai:Int) {
        this.jalan = nilai
    }

    fun getJalan(): Int {
        return jalan
    }
    fun setRodaDua(nilai:Int) {
        this.rodaDua = nilai
    }

    fun getRodaDua(): Int {
        return rodaDua
    }
    fun setRodaEmpat(nilai:Int) {
        this.rodaEmpat = nilai
    }

    fun getRodaEmpat(): Int {
        return rodaEmpat
    }
    fun setBus(nilai:Int) {
        this.bus = nilai
    }

    fun getBus(): Int {
        return bus
    }
    fun setJumlahKendaraan(total:Int) {
        this.jumlahKendaraan = total
    }

    fun getJumlahKendaraan(): Int {
        return jumlahKendaraan
    }
    fun setJumlahOrang(total:Int) {
        this.jumlahOrang = total
    }

    fun getJumlahOrang(): Int {
        return jumlahOrang
    }
    fun setWisdom(total:Int) {
        this.wisdom = total
    }

    fun getWisdom(): Int {
        return wisdom
    }
    fun setWisman(total:Int) {
        this.wisman = total
    }

    fun getWisman(): Int {
        return wisman
    }
    fun setTimestamp(nilai:String) {
        this.timestamp = nilai
    }

    fun getTimestamp(): String {
        return timestamp
    }
    fun setTaxBanyuwangi(nilai:String) {
        this.taxbanyuwangi = nilai
    }

    fun getTaxBanyuwangi(): String {
        return taxbanyuwangi
    }
}